tests/src/Kernel/fixtures/markup_rendering_patterns/banner.yml

```diff
@@ -17,9 +17,42 @@ default_page_banner_centered_with_link_full_widht:
   assertions:
     count:
       'div[data-drupal-selector=banner]': 1
-      'div.bg-light.text-dark.bcl-banner': 1
+      'div.bg-lighter.text-dark.bcl-banner': 1
       'div.full-width': 1
       'div.text-center': 1
+      'div.bcl-banner__content': 1
+      'a': 1
+      'div.hero': 0
+      'div.bcl-banner__image': 0
+    contains:
+      'bcl-banner__content': "EU Budget for the future"
+      'p': "Innovation, economy, environment and geopolitics"
+    equals:
+      'a': "Subscribe"
```

Comment:
This is way too generic, we need to assert that we are rendering using the correct bcl template. At least something like `.bcl-banner__content a[href="https://example.com"]: Subscribe`.